### PR TITLE
Fix SonnarCloud quality gate

### PR DESCRIPTION
**Pull request by Tobianas**
LIGHTY-79

### CODE REVIEW

File: lighty-applications/lighty-app-modules-config/src/main/java/io/lighty/applications/util/ModulesConfig.java [hunk 1]
```diff
@@ -80,9 +80,6 @@ public boolean equals(final Object obj) {
         if (!(obj instanceof ModulesConfig)) {
             return false;
         }
-        if (!super.equals(obj)) {
-            return false;
```
Comment (PeterSuna) [inline lighty-applications/lighty-app-modules-config/src/main/java/io/lighty/applications/util/ModulesConfig.java:84]: Please remove `super` also from hash code method. Rename commit header to "ModulesConfig reference should not be compared" or something similar.

File: lighty-tests-report/pom.xml [hunk 2]
```diff
@@ -97,6 +97,10 @@
             <groupId>io.lighty.kit.examples.controllers</groupId>
             <artifactId>lighty-guice-app</artifactId>
         </dependency>
+        <dependency>
```
Comment (PeterSuna) [inline lighty-tests-report/pom.xml:100]: This commit is about adding lighty-community-restconf-actions-app to lighty-test-report. Not about
`Add lighty-community-restconf-actions-app to lighty-bom`

**Review by PeterSuna — CHANGES_REQUESTED** [review]

File: lighty-applications/lighty-app-modules-config/src/main/java/io/lighty/applications/util/ModulesConfig.java (hunk 1 above)
Comment (Tobianas) [reply lighty-applications/lighty-app-modules-config/src/main/java/io/lighty/applications/util/ModulesConfig.java:84]: super changed to `this`

File: lighty-tests-report/pom.xml (hunk 2 above)
Comment (Tobianas) [reply lighty-tests-report/pom.xml:100]: placed to a separate commit

File: lighty-applications/lighty-app-modules-config/src/main/java/io/lighty/applications/util/ModulesConfig.java (hunk 1 above)
Comment (ihrasko) [reply lighty-applications/lighty-app-modules-config/src/main/java/io/lighty/applications/util/ModulesConfig.java:84]: I do not see `this` at all here :D

**Review by ihrasko — CHANGES_REQUESTED** [review]

File: lighty-applications/lighty-app-modules-config/src/main/java/io/lighty/applications/util/ModulesConfig.java (hunk 1 above)
Comment (Tobianas) [reply lighty-applications/lighty-app-modules-config/src/main/java/io/lighty/applications/util/ModulesConfig.java:84]: removed

File: lighty-applications/lighty-app-modules-config/src/test/java/io/lighty/applications/util/ModulesConfigTest.java [hunk 3]
```diff
@@ -8,18 +8,30 @@
...
+        SAMPLE_CONFIG.setModuleTimeoutSeconds(180);
+    }
+
     @Test
     void loadJsonConfig() throws ConfigurationException {
         final var config = ModulesConfig.getModulesConfig(this.getClass().getClassLoader()
                 .getResourceAsStream("sampleModulesConfig.json"));
 
         assertEquals(config.getModuleTimeoutSeconds(), 180);
+        assertTrue(config.equals(SAMPLE_CONFIG));
```
[the remark this answers is not in the document]
Comment (Tobianas) [reply lighty-applications/lighty-app-modules-config/src/test/java/io/lighty/applications/util/ModulesConfigTest.java]: separated

**Review by ihrasko — APPROVED** [review]

**Pull request merged**